internal/statestore/public.go

```diff
@@ -50,6 +51,12 @@ type Service interface {
 	// GetAssignments returns the assignment associated with the input ticket id
 	GetAssignments(ctx context.Context, id string, callback func(*pb.Assignment) error) error
 
+	// GetProposedTickets returns the tickets stored under the proposed sorted set withinin TTL period.
+	GetProposedTickets(ctx context.Context, ttl time.Duration) ([]string, error)
```

Comment:
Not needed - the fact that the tickets are ignored can just be internal detail of filter tickets.